cli/cmd/update.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/odigos-io/odigos/cli/cmd/resources"
+	cmdcontext "github.com/odigos-io/odigos/cli/pkg/cmd_context"
+	"github.com/odigos-io/odigos/cli/pkg/kube"
+	"github.com/odigos-io/odigos/k8sutils/pkg/consts"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// updateTokenCmd represents the update-token command
+var updateTokenCmd = &cobra.Command{
+	Use:   "update-token",
+	Short: "Update a client's token for Odigos",
+	Long: `Update the token used for authenticating with Odigos.
+This command is useful for updating your on-prem or cloud token for an existing installation.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := cmd.Context()
+		client := cmdcontext.KubeClientFromContextOrExit(ctx)
+		ns, err := resources.GetOdigosNamespace(client, ctx)
+		if err != nil {
```

Comment:
In other cli command we differentiate between:
- odigos is not installed in the cluster
- some general error (permissions, network etc)

I think it makes sense to add it here to give users a usable message